Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -141,9 +141,14 @@ internal enum PropertiesState {
 #endif
             }
 
-            if (needsClientSync) {
+            if (needsUserScriptCallback) {
+                MeshSyncLogger.VerboseLog("Sending changes, needed user script callback.");
+                needsUserScriptCallback = false;
+                sendChanges             = true;
+                m_server.RequestUserScriptCallback();
```

Comment:
Does this set `needsUserScriptCallback` to false, and then immediately call `RequestUserScriptCallback` which sets it back to true again?

Reply:
No, this tells the C++ server to send a message to the client to call a user script.

Reply:
Could you please remind me if there is a diagram somewhere that shows the flow of this user script callback ?

Reply:
There's no diagram, this just sends the "user_script_callback" message from the server to the client, which triggers the blender addon to call any methods added to "mesh_sync_server_requested_callback" in the addon. Users can add their own calls to that using an addon or python code in a blend file.